README.md

```diff
@@ -76,3 +79,41 @@ Key features of the payment app:
 As a current important implementation detail, we accept transactions from users without sufficient funds.
 In such cases, it's the sequencer's responsibility for including these transactions in the batch.
 This may change in the future.
+
+
+
+## `tripa` service
+
+Tripa is a sequencer implementation using the Paio SDK.
+It is a centralized sequencer that submits transactions to Ethereum as calldata.
+
+It exposes the following endpoints:
+* `GET /nonce`: get user nonce.
+* `GET /domain`: get the domain.
+* `GET /gas`: get gas price.
+* `POST /transaction`: post a transaction
+* `GET /batch`: get current batch
```

Comment:
Could we get a description of parameters and return value for these?

Reply:
These ones are harder because I didn't write this part of the code. I added a longer description for `/transaction`, but then I realized it may be completely wrong...

I don't think I can contribute more here.